ModularBungalow/clothing/coat.dm

```diff
@@ -99,7 +99,7 @@
 	inhand_icon_state = "armor"
 	dog_fashion = null
 
-/obj/item/clothing/suit/CoCoat
+/obj/item/clothing/suit/armor/centcom/CoCoat
```

Comment:
better idea, make it " /obj/item/clothing/suit/armor/captain/CoCoat " so it uses cap armor values. The armor centcom would be the buggy old green cap hardsuit thing

Reply:
Okay i can switch it to that no problem.